app/cards/[id]/page.tsx

```diff
@@ -33,20 +85,82 @@ export default function CardItem() {
         <h2>{calculateAge(card.dateOfBirth)}</h2>
         <h2>{card?.gender}</h2>
       </section>
-      <h2 className="mt-10 text-grey ">Вкажіть вагу та зріст Вашої дитини</h2>
-
-      <form className="flex-col mt-10">
-        <Input type="number" name="weight" placeholder="Введіть вагу в кг" />
-
-        <Input type="number" name="heigh" placeholder="Введіть зріст в см" />
+      <h2 className="mt-10 text-grey ">
+        Вкажіть вагу та зріст Вашої дитини по місяцям для побудови графіку
+      </h2>
+      <form className="flex-row mt-10" onSubmit={formik.handleSubmit}>
+        <Input
+          type="number"
+          name="age"
+          value={formik.values.age}
+          placeholder="Введіть вік в місяцях"
+          onChange={formik.handleChange}
+        />
```

Comment:
I suggest you to use `formik.getFieldProps`, it'll pass all necessary props down to the component (`name`, `value`, `onChange`, `onBlur`...)
```suggestion
<Input
  type="number"
  placeholder="Введіть вік в місяцях"
  {...formik.getFieldProps("age")}
/>
```